client/components/ui/data-table/components/cells/read-only-cell.tsx

```
@@ -0,0 +1,49 @@
+        return <>{rendered}</>;
+    }
+
+    return (
+        <div
+            onClick={onClick}
+            className={cn(
+                "cursor-pointer rounded px-2 py-1 -mx-2 -my-1",
+                "hover:bg-muted/50 hover:ring-1 hover:ring-ring/20",
+                "transition-colors duration-150 w-full min-h-[1.5rem]"
+            )}
+            role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)